src/install.rs

```diff
@@ -233,6 +236,53 @@ fn write_platform(mountpoint: &Path, platform: &str) -> Result<()> {
     Ok(())
 }
 
+/// Copy networking config if asked to do so
+fn copy_network_config(mountpoint: &Path, net_config_src: &str) -> Result<()> {
+    eprintln!("Copying networking configuration from {}", net_config_src);
+
+    // get the path to the destination directory
+    let mut net_config_dest = mountpoint.to_path_buf();
+    net_config_dest.push("coreos-installer-network");
+
+    // make the directory if it doesn't exist
+    create_dir_all(&net_config_dest)
+        .chain_err(|| {
+            format!(
+                "creating destination networking config directory {}",
+                net_config_dest.display()
```

Comment:
I don't think you need the explicit `.display()` here.

Reply:
ok, I'll check. thanks!

Reply:
looks like we do need it:

```
error[E0277]: `std::path::PathBuf` doesn't implement `std::fmt::Display`
   --> src/install.rs:251:17
    |
251 |                 net_config_dest
    |                 ^^^^^^^^^^^^^^^ `std::path::PathBuf` cannot be formatted with the default formatter
    |
    = help: the trait `std::fmt::Display` is not implemented for `std::path::PathBuf`
```